superset-frontend/src/components/ErrorMessage/ParameterErrorMessage.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { t } from '@superset-ui/core';
+
+import { ErrorMessageComponentProps } from './types';
+import IssueCode from './IssueCode';
+import ErrorAlert from './ErrorAlert';
+
+interface ParameterErrorExtra {
+  owners?: string[];
+  issue_codes: {
+    code: number;
+    message: string;
+  }[];
+  engine_name: string | null;
```

Comment:
I don't see `owners` or `engine_name` being used in the renderer, so we probably don't need them in the `Extra` section?

Reply:
It makes sense that we wouldn't have either of them too I think, since this error message comes from SQL Lab, where ownership is kinda fuzzy, and the error is returned before the query actually gets run, so the engine doesn't matter

Reply:
Right, I copied this from another component and missed these, let me fix it.

Reply:
@etr2460 right now the message is very simple, but we could return here not just the undefined parameters (which we currently do) but also the parameters provided by the user. This will allow us to show more helpful messages, eg, "You did not define the parameter `treshold`, did you mean `threshold`?" by comparing the two sets.